Readme.md

```diff
@@ -80,7 +80,7 @@ app.use((ctx, next) => {
 
 ### GeneratorFunction
 
-To use generator functions, you must use a wrapper such as [co](https://github.com/tj/co) that is no longer supplied with Koa.
+To use generator functions, you must use a wrapper such as [co](https://github.com/tj/co).
```

Comment:
I went ahead and restored the Dependency changes section I mistakenly removed.  I don't know if its meaningful, I'm not actually a v1.x user, I was attracted by the v2.x release.